.github/main.workflow

```diff
@@ -0,0 +1,47 @@
+workflow "Update data sources" {
+  on = "schedule(0 09 * * *)"
+  resolves = ["Fetch latest data sources"]
+}
+
+action "Fetch latest data sources" {
+  uses = "actions/npm@master"
+  args = "run update-data-sources"
+  secrets = [
+    "GITHUB_TOKEN",
+  ]
+}
+
+workflow "Test and publish" {
+  on = "push"
+  resolves = ["Publish via semantic-release", "Upload Algolia Indices"]
+}
+
+action "Install dependencies" {
+  uses = "actions/npm@master"
+  args = "ci"
+}
+
+action "Run tests" {
+  uses = "actions/npm@master"
+  needs = ["Install dependencies"]
+  args = "test"
+}
+
+action "Filters for GitHub Actions" {
```

Comment:
Can we rename this so that it's less confusing in the GitHub checks UI? 🤔 

```suggestion
action "Only publish master branch" {
```

or similar

Reply:
1c4c580dc36fcc43094200ce662c6556aff46782 and ccfb270bc75104d2a69567ac0c70e8aef6fc6481